docs/api.rst

```diff
@@ -3192,7 +3193,11 @@ Search for assets(Levenshtein)
    :reqjson list[bool] ascending: Should the order be ascending? This is the default. If set to false, it will be on descending order.
```

Comment:
You removed the filter in this endpoint but you didn't update the docs.

For the record. Alexey checked and we were not using pagination on this endpoint, only the limit attribute so he kept it in the code